presto-thrift-api/src/main/java/com/facebook/presto/thrift/api/udf/ThriftUdfPage.java

```diff
@@ -24,21 +25,25 @@
 import java.util.Objects;
 
 import static com.facebook.drift.annotations.ThriftField.Requiredness.OPTIONAL;
+import static com.facebook.presto.thrift.api.udf.ThriftUdfPageFormat.PRESTO_SERIALIZED;
 import static com.facebook.presto.thrift.api.udf.ThriftUdfPageFormat.PRESTO_THRIFT;
 
 @ThriftStruct("UdfPage")
 public class ThriftUdfPage
 {
     private final ThriftUdfPageFormat pageFormat;
     private final List<PrestoThriftBlock> thriftBlocks;
+    private final ThriftSerializedPage prestoPage;
```

Comment:
It should be possible to make it `Optional`

Reply:
I didn't see any of the thrift structs using `Optional` though.